src/docs/get-started/install/_path-linux-chromeos.md

```diff
@@ -10,19 +10,19 @@ all terminal sessions are machine-specific.
 Typically you add a line to a file that is executed
 whenever you open a new window. For example:
 
- 1. Determine the directory where you placed the Flutter SDK.
+ 1. Determine the path of your clone of the Flutter SDK.
```

Comment:
Note that the user may have downloaded the prepackaged archive, and not cloned it themselves.

Reply:
Ah that's a good point, the surrounding directions refer to it as a git clone, but we should probably fix that. (go for it!)